swagger.yaml

```diff
@@ -2820,36 +2882,78 @@ paths:
 definitions:
   operator:
     type: object
+    required:
+      - name
+      - operator_type
     properties:
+      name:
+        type: string
+        description: Unique name of the operator.
       operator_type:
-        $ref: '#/definitions/operator-type'
-      enabled:
-        type: boolean
-        default: false
+        type: string
+        enum: ['built-in', 'custom']
```

Comment:
custom ? why not olm?

Reply:
In the same way no other specific operator name wasn't used. IMO it is a more abstract definition.

Reply:
Changed to OLM.